docs/how_to/01-placeholders.rst

```diff
@@ -1,34 +1,44 @@
 .. _placeholders_outside_cms:
 
-#######################################
 How to use placeholders outside the CMS
-#######################################
+=======================================
 
-Placeholders are special model fields that django CMS uses to render
-user-editable content (plugins) in templates. That is, it's the place where a
-user can add text, video or any other plugin to a webpage, using the same
-frontend editing as the CMS pages.
+Placeholders are special model fields that django CMS uses to render user-editable
```

Comment:
I wonder about the crossover between `Placeholder` in a template and `PlaceholderField` in a model.

Perhaps this should start "Placeholder fields are special..."